src/site/includes/detail-page.html

```diff
@@ -20,10 +20,18 @@
         {% include "src/site/components/navigation-sidebar-trigger.html" with menuId = "va-detailpage-sidebar" %}
       {% endif %}  
       
+    {% if no_article_bottom_padding === true %}
+      <article class="usa-content vads-u-padding-bottom--0">
+        <h1>{{ heading }}</h1>
+        {{ contents }}
+      </article>
+    {% else %}  
       <article class="usa-content">
         <h1>{{ heading }}</h1>
         {{ contents }}
       </article>
+    {% endif %}
+
```

Comment:
Liquid actually doesn't have triple equals. I think this evaluates to `true` every time, so I think you'll need to switch to double equals.

Reply:
Thanks! I removed `== true`